app/views/issues/show.html.erb

```diff
@@ -15,18 +15,26 @@
 </ul>
 
 <ul class="nav nav-tabs main-tabs">
-  <li class="active">
+  <li <%= "class=active" if !params[:tab] || params[:tab] == 'info-tab' %>>
```

Comment:
```suggestion
  <li<%= " class=active" if !params[:tab] || params[:tab] == 'info-tab' %>>
```

Reply:
^ silly suggestion to avoid extra white space in out generated html:
<img width="576" alt="Captura de pantalla 2019-09-27 a las 11 23 55" src="https://user-images.githubusercontent.com/85766/65762797-ba725d80-e119-11e9-9f46-d15cd3c9d80b.png">